eng/Subsets.props

```diff
@@ -326,6 +328,31 @@
       Test="true" Category="clr" Condition="'$(DotNetBuildFromSource)' != 'true' and '$(NativeAotSupported)' == 'true'"/>
   </ItemGroup>
 
+  <ItemGroup Condition="$(_subset.Contains('+tools.linker+'))">
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\linker\Mono.Linker.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\ILLink.Tasks\ILLink.Tasks.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\analyzer\analyzer.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\ILLink.RoslynAnalyzer\ILLink.RoslynAnalyzer.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\linker\ref\Mono.Linker.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\tlens\tlens.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\ILLink.CodeFix\ILLink.CodeFixProvider.csproj" Category="tools" />
+  </ItemGroup>
+
+  <ItemGroup Condition="$(_subset.Contains('+tools.linkertests+'))">
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\test\Mono.Linker.Tests\Mono.Linker.Tests.csproj"
```

Comment:
These should be built even in source build.

Reply:
Ah, never mind, I guess we don't need tests for source build.

Reply:
In a nutshell source-build is a requirement from some Linux distributions which require source code to build. So basically when they get dotnet they actually get all the source code (one of those pieces is the runtime, but they are more pieces) and build it. The source-build repository is in charge of taking all of these pieces and glue them together, it also has an explanation of what source build is -> https://github.com/dotnet/source-build#source-build-goals
We want all the source code to be included, but the tests are not going to be executed once it's an app like dotnet so we exclude them using the DotNetBuildFromSource variable.